src/semanticsEditorWrapper.js

```diff
@@ -1,42 +1,25 @@
 /* eslint-env browser */
+/* global Vue */
```

Comment:
In fact, it looks like a lot of the code here is duplicated. Can this file actually be removed now?